srcpkgs/spotify/template

```diff
@@ -11,8 +11,8 @@ short_desc="Proprietary music streaming client"
 maintainer="Stefan Mühlinghaus <jazzman@alphabreed.com>"
 license="custom:Proprietary"
 homepage="https://www.spotify.com"
-distfiles="http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_${version}.${_subver}_amd64.deb"
```

Comment:
don't hardcode, use the variables above

Reply:
it gives error 
http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_1.2.26.1187.g36b715a1_amd64.deb: Not Found
=> ERROR: spotify-1.2.26_1: failed to fetch 'spotify-client_1.2.26.1187.g36b715a1_amd64.deb'.

Reply:
update the variables instead of removing them and replacing them with new values.

Reply:
fixed thanks